### PR TITLE
Feature/about section

### PR DESCRIPTION
**Pull request by OrenMoveo**
# About Section Feature

## This PR Includes:
- About section implementation for both mobile and desktop.

## Notes:
- When the font sizes inside the app store's text did not fit the container, I had to make some design decisions.

## Example:

https://github.com/OrenMoveo/Epicure/assets/156906187/4237d501-0978-4669-ad5d-0e7d875aea55

### CODE REVIEW

File: src/components/AboutSection/AboutSection.scss [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          display: flex;
+          flex-direction: row-reverse;
+          justify-content: flex-end;
+          align-items: flex-start;
+          order: 2;
+          padding-bottom: 0;
+        }
+
+        .apple-store-container,
+        .google-play-store-container {
```
Comment (nitzanpap) [inline src/components/AboutSection/AboutSection.scss:90]:
```scss
.google-play-store-icon-container
```
Would be a better name. Your name could mean different things ( e.g. a docker container that wraps a google play store service ).
Same goes for all other places where it is not clear.

File: src/components/AboutSection/AboutSection.scss [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      }
+
+      .app-stores-container {
+        $app-stores-border-width: 0.5px;
+        $app-stores-border-color: rgba(0, 0, 0, 1);
+        $app-stores-border-texture: solid;
+        $app-stores-border-radius: 10px;
+        $app-stores-container-horizontal-padding: 12px;
+        $app-stores-container-vertical-padding: 7px;
+        $app-stores-container-gap: 7px;
```
Comment (nitzanpap) [inline src/components/AboutSection/AboutSection.scss:70]: You are only using these variables once and they mean a very specific thing. Do you really need them?
If not, remove them.

File: src/components/AboutSection/AboutSection.scss [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+          width: 665px;
+          height: 35px;
+          font-size: 30px;
+          line-height: 35px;
+          letter-spacing: 2.140000104904175px;
+          @include desktop {
+            padding-bottom: 0;
+          }
+        }
+      }
```
Comment (nitzanpap) [inline src/components/AboutSection/AboutSection.scss:168]: Remove redundant `@include desktop`.
Did you mean  `@include tablet`?
Please fix.

**Review by nitzanpap — CHANGES_REQUESTED** [review]

**Comment by nitzanpap** [comment]
Also note that you UI breaks when the screen width is between 820px and 960px.
Not urgent but It will come up on the pixel perfect tasks.

File: src/components/AboutSection/AboutSection.scss (hunk 1 above)
Comment (OrenMoveo) [reply src/components/AboutSection/AboutSection.scss:90]: The name represents the entire container and not just the icon:
Screenshot 2024-01-25 at 17 23 55
Should i make the change?
@nitzanpap

Comment (nitzanpap) [reply src/components/AboutSection/AboutSection.scss:90]: That is why it should be named google-play-store-icon-**container**

**Pull request merged**